lib/util/SparqlExpressionEvaluator.js

```diff
@@ -23,7 +50,7 @@ function SparqlExpressionEvaluator(expression) {
   if (!expression) return noop;
   var expressionType = expression && expression.type || typeof expression,
       evaluator = evaluators[expressionType];
-  if (!evaluator) throw new UnsupportedExpressionError(expressionType);
+  if (!evaluator) throw new Error('Unsupported expression type: ' + expressionType);
```

Comment:
There isn't one, this was probably reverted by copy paste, since this branch existed  before https://github.com/LinkedDataFragments/Client.js/commit/73e690b3480784b9d13ef85634da1bdf1496c4ea